invoice/src/main/java/org/killbill/billing/invoice/dao/DefaultInvoiceDao.java

```diff
@@ -836,7 +837,7 @@ public InvoicePaymentModelDao postChargebackReversal(final UUID paymentId, final
                                         invoicePayment.getProcessedCurrency(),
                                         invoicePayment.getPaymentCookieId(),
                                         invoicePayment.getLinkedInvoicePaymentId() == null ? null : invoicePayment.getLinkedInvoicePaymentId().toString(),
-                                        false,
+                                        InvoicePaymentStatus.INIT.toString(),
```

Comment:
This is where a fourth state could have been useful: when the chargeback is reversed, it's not really _INIT_ (and it's not really _FAILED_ either, see https://github.com/killbill/killbill/issues/477#issuecomment-220989853).

This is such a rare case though, not worth optimizing.